nemo/collections/tts/torch/data.py

```diff
@@ -609,6 +644,8 @@ def general_collate_fn(self, batch):
             "align_prior_matrix": align_prior_matrices if AlignPriorMatrix in self.sup_data_types_set else None,
             "pitch": torch.stack(pitches) if Pitch in self.sup_data_types_set else None,
             "pitch_lens": torch.stack(pitches_lengths) if Pitch in self.sup_data_types_set else None,
+            "voiced_mask": torch.stack(voiced_masks) if Pitch in self.sup_data_types_set else None,
+            "p_voiced": torch.stack(p_voiceds) if Pitch in self.sup_data_types_set else None,
```

Comment:
Let's use the actual data type and not `Pitch`

Reply:
Done